docs/concepts/configuration/assign-pod-node.md

```diff
@@ -298,231 +298,7 @@ Highly Available database statefulset has one master and three replicas, one may
-some characteristic(s) of the pod to determine that the pod should be allowed to use
-the special nodes and hence the admission controller should add the toleration.
-To ensure that the pods that need
-the special hardware *only* schedule onto the nodes that have the special hardware, you will need some
-additional mechanism, e.g. you could represent the special resource using
-[opaque integer resources](/docs/concepts/configuration/manage-compute-resources-container/#opaque-integer-resources-alpha-feature)
-and request it as a resource in the PodSpec, or you could label the nodes that have
-the special hardware and use node affinity on the pods that need the hardware.
-
-* **per-pod-configurable eviction behavior when there are node problems (alpha feature)**,
-which is described in the next section.
-
-### Per-pod-configurable eviction behavior when there are node problems (alpha feature)
-
-Earlier we mentioned the `NoExecute` taint effect, which affects pods that are already
-running on the node as follows
-
- * pods that do not tolerate the taint are evicted immediately
- * pods that tolerate the taint without specifying `tolerationSeconds` in
-   their toleration specification remain bound forever
- * pods that tolerate the taint with a specified `tolerationSeconds` remain
-   bound for the specified amount of time
-
-The above behavior is a beta feature. In addition, Kubernetes 1.6 has alpha
-support for representing node problems (currently only "node unreachable" and
-"node not ready", corresponding to the NodeCondition "Ready" being "Unknown" or
-"False" respectively) as taints. When the `TaintBasedEvictions` alpha feature
-is enabled (you can do this by including `TaintBasedEvictions=true` in `--feature-gates`, such as
-`--feature-gates=FooBar=true,TaintBasedEvictions=true`), the taints are automatically
-added by the NodeController and the normal logic for evicting pods from nodes
-based on the Ready NodeCondition is disabled.
-(Note: To maintain the existing [rate limiting](/docs/concepts/architecture/nodes/)
-behavior of pod evictions due to node problems, the system actually adds the taints
-in a rate-limited way. This prevents massive pod evictions in scenarios such
-as the master becoming partitioned from the nodes.)
-This alpha feature, in combination with `tolerationSeconds`, allows a pod
-to specify how long it should stay bound to a node that has one or both of these problems.
-
-For example, an application with a lot of local state might want to stay
-bound to node for a long time in the event of network partition, in the hope
-that the partition will recover and thus the pod eviction can be avoided.
-The toleration the pod would use in that case would look like
-
-```yaml
-tolerations:
-- key: "node.alpha.kubernetes.io/unreachable"
-  operator: "Exists"
-  effect: "NoExecute"
-  tolerationSeconds: 6000
-```
-
-(For the node not ready case, change the key to `node.alpha.kubernetes.io/notReady`.)
```

Comment:
To reviewers: this line was too specific in the context of more taints supported, so removed in revision.